### ISSUE TITLE
Types for ethermint transaction decoding?

### ISSUE THREAD

**Issue by Undead8** [cosmos/cosmjs#1060]
I need to decode transactions of type `'/ethermint.evm.v1.MsgEthereumTx'`.

I am able to decode the transaction itself, but in order to decode the messages I understand that I must use a registry. My issue is that the `defaultRegistryTypes` does not contain the `'/ethermint.evm.v1.MsgEthereumTx'` type, so I am unable to decode the messages with the default registry. 

Could you kindly point me in the right direction as to how I could include this type and where I could find it?

@webmaster128 

Thanks a lot for your help!

**Comment by webmaster128** [comment]
Right. You need to generate the TS code for those types. Or someone from the Ethermint team can do that for you. See https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md for how to do the code generation.

**Comment by giansalex** [comment]
append this issue `Unsupported type: '/ethermint.types.v1.EthAccount'.` when i try to sign, 
> keplr has ethermint support in `v0.9.14`
